app/bundles/PluginBundle/Model/PluginModel.php

```diff
@@ -62,7 +70,10 @@ public function getPermissionBase()
      */
     public function getLeadFields()
     {
-        return $this->leadFieldModel->getFieldList();
+        $leadFields         =  $this->leadFieldModel->getFieldList();
+        $leadFields['Core'] = ['id' => $this->translator->trans('mautic.lead.report.contact_id')] + $leadFields['Core'];
```

Comment:
Hello @kuzmany sometimes i have an issue but it is not reproducable all the time. It throws a 500.

>[2018-03-12 11:13:47] mautic.ERROR: Symfony\Component\Debug\Exception\FatalThrowableError: Unsupported operand types - in file /home/path/app/bundles/PluginBundle/Model/PluginModel.php - at line 74 [] []

It seems that the `+` at that line generates the issue. But we couldn't reproduce all the time... I don't know what to do :)

Reply:
We found something !

It is working in english but not in other languages. We're having a look on it and we comeback to you :)

Reply:
@kuzmany if you change `line 74` by the following code, it solves the issue :)
```php
$leadFields[$this->translator->trans('mautic.lead.field.group.core')]['id'] = $this->translator->trans('mautic.lead.report.contact_id');
```

Reply:
Fixed

Reply:
Fixed